libraries/chain/include/eosio/chain/backing_store.hpp

```diff
@@ -0,0 +1,34 @@
+#pragma once
+
+#include <fc/reflect/variant.hpp>
+#include <stdexcept>
+
+namespace eosio { namespace chain {
+
+   enum class backing_store_type {
+      NATIVE, // A name for regular users. Uses Chainbase.
+      ROCKSDB
```

Comment:
We shouldn't be calling this `ROCKSDB` either, it's not consistent with the naming used everywhere else in your code. `CHAIN_KV` would be a better fit.

Reply:
EOS is all about mixed metaphors, but I thought the idea was that chain_kv could be designed to work with several databases.  Right now it may be tied all together, but eventually, chain_kv could be loaded for any number of databases, but this backing store will determine which type of database you are storing in.

Reply:
Done